spec/models/link_history_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe LinkHistory, type: :model do
+  describe "#add_errors" do
+    let(:link) { FactoryGirl.create(:link) }
+    let(:link_history) { FactoryGirl.create(:link_history, link: link) }
```

Comment:
We could move the `link: link` part here into the factory so we don't need to explicitly create a link.